frontend_app_thread/src/index.dev.js

```diff
@@ -2,22 +2,22 @@ import React from 'react'
 import 'core-js/stable'
 import 'regenerator-runtime/runtime'
 import ReactDOM from 'react-dom'
-// import Thread from './container/Thread.jsx'
+import Thread from './container/Thread.jsx'
 import { debug } from './debug.js'
 import { LiveMessageManager } from 'tracim_frontend_lib'
-import PopupCreateThread from './container/PopupCreateThread.jsx'
+// import PopupCreateThread from './container/PopupCreateThread.jsx'
```

Comment:
Maybe remove this line?

Reply:
I chose to keep it because there is no way of switching between the components PopupCreateThread and Thread other than commenting one or the other in the ReactDOM.render().

It's not the cleanest but it's something that was here before my refactor

Reply:
won't fix